homeassistant/components/homewizard/sensor.py

```diff
@@ -108,13 +108,28 @@ class HomeWizardSensorEntityDescription(SensorEntityDescription):
         has_fn=lambda data: data.total_energy_import_kwh is not None,
         value_fn=lambda data: data.total_energy_import_kwh,
     ),
+    HomeWizardSensorEntityDescription(
+        key="total_power_import_t1_kwh",
+        translation_key="total_energy_import_kwh",
+        native_unit_of_measurement=UnitOfEnergy.KILO_WATT_HOUR,
+        device_class=SensorDeviceClass.ENERGY,
+        state_class=SensorStateClass.TOTAL_INCREASING,
+        has_fn=lambda data: (  # HWE-SKT provides tarif 1 only
+            data.total_energy_import_t1_kwh is not None
+            and data.total_energy_import_kwh is None
+        ),
+        value_fn=lambda data: data.total_energy_import_t1_kwh,
+    ),
```

Comment:
This should already have been fixed in [python-homewizard-energy-3.1.0](https://github.com/homewizard/python-homewizard-energy/releases/tag/v3.1.0). And when I test this in 'real Home Assistant' (not pytest) it seems to work just fine, this extra sensor can be removed.

```suggestion
```

Reply:
This is why: #104537

Reply:
Ah, makes sense!

Reply:
Fixed it during the rebase onto #104537 (as it caused merge conflicts/test conflicts)